src/components/List.js

```diff
@@ -36,29 +45,47 @@ const List = ({ token }) => {
     <div>
       <h1>List</h1>
 
-      {docs.length === 0 && (
+      {docs.length === 0 ? (
         <section>
           <p>Your shopping list is currently empty.</p>
           <a href="/add-item">Add an Item</a>
         </section>
+      ) : (
+        <form>
+          <label for="search-bar">Filter Items</label>
```

Comment:
I think this needs to be `htmlFor`? `for` is reserved in JavaScript

Reply:
You are correct, will make those changes.